src/maxKey.js

```diff
@@ -8,13 +8,33 @@
  * @param {object} array - A non-empty array of numbers
  * @returns {number} The key with the largest value
  */
-function maxKey(obj) {
-  // This is your job. :)
+
+ let a = 22
+let b = 44
+let c = 13
+let d = 3
+
+function maxKey(array) {
```

Comment:
The input to this function isn't an `array`, so this isn't a good variable name. The functions as I've defined them are correct; you don't need to change anything about what they accept as input.